modules/smartgcal/src/main/scala/lucuma/odb/smartgcal/data/Flamingos2.scala

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.odb.smartgcal.data
+
+import cats.data.NonEmptyList
+import cats.syntax.option.*
+import eu.timepit.refined.types.numeric.PosInt
+import eu.timepit.refined.types.numeric.PosLong
+import fs2.Pipe
+import fs2.Stream
+import lucuma.core.enums.F2Disperser
+import lucuma.core.enums.F2Filter
+import lucuma.core.enums.F2Fpu
+import lucuma.core.util.TimeSpan
+import lucuma.odb.smartgcal.data.SmartGcalValue.LegacyInstrumentConfig
+import monocle.Focus
+import monocle.Lens
+import monocle.Optional
+import lucuma.core.model.sequence.f2.F2DynamicConfig
+
+object Flamingos2:
+
+  case class SearchKey(
```

Comment:
This is taken from Gmos, I'm not sure if I need a separate `SearchKey` and `TableKey`

Reply:
I'm struggling to remember why I thought I needed distinct types for GMOS.  It may be that the search key once separated the wavelength.  I'm not sure.  If you don't see a need for the distinction by all means merge them.